app/controllers/katello/api/v2/organizations_controller.rb

```diff
@@ -62,6 +62,7 @@ def show
     param :redhat_repository_url, String, :desc => N_("Red Hat CDN URL")
     param_group :resource, ::Api::V2::TaxonomiesController
     def update
+      params.delete(:users) if params[:users]  # Cannot PUT a GET result as the UI does
```

Comment:
Wondering if we should add a comment so that we can remove this line when foreman moves to strong_params.